.goreleaser.yaml

```diff
@@ -2,7 +2,16 @@ project_name: cert-manager-openshift-routes
 before:
   hooks:
     - go test -v ./...
+    # update static manifests
     - ./hack/generate-static-manifest.sh {{ .Version }}
+    # update Helm chart version
+    - sed -i 's/^version:.*$/version: {{ trimprefix .Version "v" }}/' deploy/chart/Chart.yaml
+    - sed -i 's/^appVersion:.*$/appVersion: {{ trimprefix .Version "v" }}/' deploy/chart/Chart.yaml
+    # package Helm
+    - helm package ./deploy/chart --version {{ trimprefix .Version "v"}} --app-version {{ .Version }}
+    # upload Helm chart as OCI artifact to GitHub Container Registry
+    - '{{if .IsSnapshot}}echo SKIPPING: {{end}}helm push ./openshift-routes-{{ trimprefix .Version "v"}}.tgz oci://ghcr.io/cert-manager/cert-manager-openshift-routes:{{ .Version }}-chart'
```

Comment:
Is the repo path ok or should we use a subpath (`ghcr.io/cert-manager/cert-manager-openshift-routes/chart:{{ trimprefix  .Version "v" }}`)?

Reply:
I pushed a commit and tagged it with v0.4.0-alpha.0 to test it out: https://github.com/cert-manager/openshift-routes/actions/runs/7088552754/job/19291296517.

Reply:
@jacksgt It works now! The action job [19293897798](https://github.com/cert-manager/openshift-routes/actions/runs/7088552754/job/19293897798) is all green. I haven't made the Helm chart OCI registry public yet since I'm not sure about the Helm chart name yet.

Should we go with 
```
ghcr.io/cert-manager/charts/openshift-routes
```
or
```
ghcr.io/cert-manager/charts/cert-manager-openshift-routes
```
?

Reply:
Thanks for the fix!

I'd go with `ghcr.io/cert-manager/charts/openshift-routes`, since the Helm chart itself is called `openshift-routes`: https://github.com/cert-manager/openshift-routes/blob/main/deploy/chart/Chart.yaml#L2

Reply:
Agreed. It makes sense to call the chart after the GitHub repository.

I hope that the inconsistence between the container image name and the Helm chart name won't be a problem 😅